### ISSUE TITLE
Invalid default docker version on Debian

### ISSUE THREAD

**Issue by deric** [puppetlabs/puppetlabs-kubernetes#564]
## Describe the Bug

The default value for `kubernetes::docker_version` expects certain distributions and configuration (like `manage_docker:true`).

## Expected Behavior

I'm expecting catalog to compile with default parameters.

```
Error 500 on SERVER: Server Error: Evaluation Error: The value '5:20.10.11~3-0~ubuntu-' cannot be converted to Numeric. (file: /etc/puppetlabs/code/environments/k8s/modules/kubernetes/manifests/init.pp, line: 556, column: 17)
```

## Steps to Reproduce

```puppet
  class { 'kubernetes':
    controller => true,
  }
```

## Environment
 - Version 6.3.0
 - Platform Debian 11

A workaround is not to manage docker and override `docker_version` variable to some random string, but this shouldn't be required.

```yaml
kubernetes::manage_docker: false
kubernetes::docker_version: 'foobar'
```

**Comment by deric** [comment]
Resolved by #565

**Comment by pmcmaw** [comment]
Thanks for raising this issue and also following up on closing it, now that the issue has been resolved! 
Much appreciated @deric 👍
